kotlin/internal/toolchains.bzl

```diff
@@ -174,8 +176,8 @@ _kt_toolchain = rule(
             cfg = "target",
         ),
         "jvm_target": attr.string(
-            doc = "the -jvm_target flag. This is only tested at 1.8.",
-            default = "1.8",
+            doc = "the -jvm_target flag.",
+            default = "11",
```

Comment:
I'll do this against the main branch, but there are a few more JDK releases that we can add here.

Reply:
Got tired of the warnings. Will revert.